main.md

```diff
@@ -219,24 +219,26 @@ The following is an example of such a response:
 
 ## Error Response {#error_response}
 
-### Error responses
+The authorization server responds with an error response in the same way as defined for the token endpoint in [@!RFC6749].
 
-#### Authentication required
-If the client authentication fails, the authorization server shall return `401 Unauthorized` HTTP error response.
+In addition to the error codes as defined in Section 5.2 of [@!RFC6749], the pushed authorization request endpoint will also respond with the following HTTP status codes, and, if necessary, error codes. 
 
-#### Authorization required
-If the client is not authorized to perform the authorization request it wanted to post, the authorization server shall return `403 Forbidden` HTTP error response.
+### Unsupported Response Type
+The authorization server does not support obtaining an authorization code using this method. The AS responds with HTTP status code 400 and `error` value `unsupported_response_type`.
 
-#### Invalid request
-If the request object received is invalid, the authorization server shall return `400 Bad Request` HTTP error response.
+### Invalid Scope
```

Comment:
invalid_scope is already in Section 5.2 of [@!RFC6749] FWIW

Reply:
thanks. removed it.